src/peerbook/libp2p_peer.proto

```diff
@@ -16,6 +16,7 @@ message peer {
     int64 timestamp = 5;
     map<string, association_list> associations = 6;
     bytes network_id = 7;
+    map<string, bytes> signed_metadata = 8;
```

Comment:
I see you're using binaries for the keys below. Is this correct?

Reply:
I *think* so? I'm really not sure though?

Reply:
If you’re using a binary for a key you should change the type in the proto. And add a round trip test

Reply:
I kind of want a string (in a binary) for the key and a richer way to encode the value (so we can have integer/floats/strings/binaries), I'm jut not good at protobuffs

Reply:
Pretty sure a string in a binary is still a binary in protobuf

Reply:
You can't use `bytes` as a key, so they're strings forever.